content/get-involved/index.md

```diff
@@ -4,64 +4,90 @@ date: 2018-03-20T07:41:22+13:00
-
-    <hr style="margin: 40px 0" />
-
-    <h2>Starter Projects</h2>
-    <p>Do you want to begin coding for NavCoin but don’t have a specific improvement in mind? Here are a few ideas:</p>
-
-    <h3>Fix existing issues</h3>
-    <p>The issue tracker for each repository is the best place to find a useful way to contribute to NavCoin. Before starting to write any patches for issues you find, you may want to comment on the issue to make sure nobody else is already working on it.</p>
-
-    <h3>Write tests</h3>
-    <p>NavCoin’s various softwares are covered by many tests, but patches that improve test coverage are always welcome and are a great way to build familiarity with the codebase. See the documentation about automated testing.</p>
-
-    <hr style="margin: 40px 0" />
-
-    <h2>Documentation</h2>
-    <p>If you are interested in learning more about the technical details of NavCoin and how to use existing tools and APIs, it is recommended you start by exploring the self documented code in GitHub or join the discussion on the NavCoin Discord.
-
-{{< /center_text_section >}}
-
+    btn1Link="https://github.com/NAVCoin"
+    btn1Txt="GitHub"
+    btn2Link="/roadmap"
+    btn2Txt="Roadmap"
+    fontColor="#FFF">}}
+<p>The NavCoin protocol is open-source and free for anyone to contribute. If you'd like to pitch in on existing projects take a look at the Roadmap to find one that interests you, or you could start a project of your own. You can also contribute by reviewing pull requests or writing patches.</p>
+<p>The self-documented code in GitHub is a great place to get familiar with NavCoin, or you can join the discussion on the NavCoin Discord.</p>
+{{< /left_image_section >}}
+{{< left_image_section
+    title="Content Creation"
+    id="brand-bible"
+    imgSrc="/images/icons/icon-purple-book-new.png"
+    imgHeight="300px"
+    imgPadding="0px"
+    imgMarginTop="22px"
+    bgColor="#FFF"
+    linkBgColor="#523971"
+    gradient=""
+    btn1Link="/assets/NavCoinBrandGuidelines.pdf"
+    btn1Txt="Get the guide"
+    >}}
+<p>The NavCore brand guidlines collates the NavCoin Core Content Creators suggestions on how to create a consistent look and feel for promotional material and brand assets.</p> 
+<p>It gives useful tips on how to create visuals and write with the NavCoin tone of voice. It’s hoped that having these guidelines will make it easier for you to get started on creating content for NavCoin.</p>
+{{< /left_image_section >}}
+{{< left_image_section
+    title="Organizing Events"
+    id="meetup"
+    imgSrc="/images/icons/icon-goals-purple.png"
+    imgHeight="300px"
+    imgPadding="0px"
+    imgMarginTop="13px"
+    bgColor="#7d5ab5"
+    linkBgColor="#523971"
+    gradient=""
+    btn1Link=""
+    btn1Txt="Coming Soon"
```

Comment:
We should probably just have no button here.
Just have 'Coming soon' in the text